feedTheMonster.ts

```diff
@@ -227,9 +278,15 @@ class App {
       this.handleUpdateFoundMessage();
     }
   }
-
-  private handleBeforeUnload = (event: BeforeUnloadEvent): void => {
-    FirebaseIntegration.getInstance().sendSessionEndEvent();
+  private handleVisibilityChange = () => {
+    if (document.visibilityState === "visible") {
```

Comment:
Can we use **VISIBILITY_CHANGE** from **"../common/event-names"** instead of using **"visible"** ?